main.js

```diff
@@ -0,0 +1 @@
+import './style.css';
```

Comment:
Let's move these files to some folder so it's clear they are test files, they shouldn't be in the root folder.